frontend/src/components/Pokemon/Pokemon.tsx

```diff
@@ -10,13 +11,19 @@ interface Props {
 }
 
 function Pokemon(props: Props) {
+  const [flipped, setFlipped] = useState(false);
+
   return (
     <Style.PokeBox>
       <Style.Title>
         {props.name}
       </Style.Title>
+      <Style.FlipButton>
+        <img onClick={() => setFlipped(!flipped)} src="flip.svg" alt="Flip pokemon"/>
```

Comment:
Pour information, tu as plusieurs possibilités d'appeler une fonction dans un composant React : event handlers, binding, arrow functions… 

Les arrow functions (ce que tu as fait ici) est le plus simple à lire et à écrire, mais peut parfois (sur des gros composants) amener des problèmes de perf. Je t'encourage à lire la doc [Passing Functions to Components](https://reactjs.org/docs/faq-functions.html#is-it-ok-to-use-arrow-functions-in-render-methods) et en particulier la section [Is it OK to use arrow functions in render methods?](https://reactjs.org/docs/faq-functions.html#is-it-ok-to-use-arrow-functions-in-render-methods) si tu veux en savoir plus !

Reply:
Effectivement, mon IDE me fait un warning dessus:
```
TSLint: Lambdas are forbidden in JSX attributes due to their rendering performance impact(jsx-no-lambda)
```